docs/installation/docker.md

```diff
@@ -84,3 +85,11 @@ Please execute the script itself to view all additional options:
 ```shell
 ./docker-run.sh
 ```
+
+## Known Problems & Solutions
+
+- Should you experience problems with the ElasticSearch container, try to
```

Comment:
Figured it won't hurt to include this information.

Reply:
Just a random PSA, its Elasticsearch, no capital on the search :)